src/System.IO.Pipes/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -1,4 +1,5 @@
 # Exposed public in System.IO.Pipes.AccessControl but implemented in System.IO.Pipes
+TypesMustExist : Type 'System.IO.Pipes.AnonymousPipeServerStreamAcl' does not exist in the reference but it does exist in the implementation.
```

Comment:
Is this needed? I found it by checking all the special files in the project for any mention of APIs that are only available in Windows (ACL related).

Reply:
I presume so. I'm not sure where the check happens in the build.

Reply:
Confirmed. If I don't add this line (in this PR and in the `NamedPipeServerStream` PR), I get an error like this:

```
C:\Users\carlos\.nuget\packages\microsoft.dotnet.apicompat\5.0.0-beta.19555.7\build\Microsoft.DotNet.ApiCompat.targets(143,5):
    error : TypesMustExist : Type 'System.IO.Pipes.NamedPipeServerStreamAcl' does not exist in the reference but it does exist in the implementation.
    [D:\corefx\src\System.IO.Pipes\src\System.IO.Pipes.csproj]
C:\Users\carlos\.nuget\packages\microsoft.dotnet.apicompat\5.0.0-beta.19555.7\build\Microsoft.DotNet.ApiCompat.targets(157,5):
    error : MatchingRefApiCompat failed - The reference assembly doesn't match all the APIs in the implementation for 'D:\corefx\artifacts\bin\System.IO.Pipes\netcoreapp-Windows_NT-Debug\System.IO.Pipes.dll'.
    To address either fix errors in the reference assembly (referenced as implementation in compat errors for this reverse compat check), add the issues to the baseline file 'D:\corefx\src\System.IO.Pipes\src\MatchingRefApiCompatBaseline.txt' or disable this check by setting RunMatchingRefApiCompat=false in this project. 
    [D:\corefx\src\System.IO.Pipes\src\System.IO.Pipes.csproj]
```